benchbuild/source/http.py

```diff
@@ -109,6 +109,36 @@ def version(self, target_dir: str, version: str) -> pb.LocalPath:
         return target_path
 
 
+class HTTPMultiple(HTTP):
+    """
+    Fetch and download multiple files via HTTP.
+    """
+
+    def __init__(
+        self,
+        local: str,
+        remote: tp.Union[str, tp.Dict[str, str]],
+        files: tp.List[str]
+    ):
+        super().__init__(local, remote)
+        self._files = files
+
+    def fetch_version(self, version: str) -> pb.LocalPath:
```

Comment:
Fetch will download the selected version into the cache path of benchbuild. You need to override ``version(...)`` as well to make sure your additional files are copied into the build directory of the project. The rest already looks good to me.